### DISCUSSION TITLE
The lasted RE3 Framework disables TAA

### DISCUSSION THREAD

**Discussion by ExcessiveGBH** [praydog/REFramework#388] [General Questions] [answered]
The lasted RE3 Framework disables TAA, tried the TAA jitter setting but made no difference.
Location of the donut shop out in the street area; shopping trolleys have really bad crawling aliasing.

**Comment by praydog** [comment, marked as answer]
Delete the `re2_sharpness_removal.lua` from your `reframework/autorun` folder.

**Reply by ExcessiveGBH** [reply]
Deleted the re2_sharpness_removal.lua.  TAA is still being disabled. 

Sys spec Win 11, Strix 3090, x299  i9 10920X.

**Reply by praydog** [reply]
Delete any reframework or autorun folder you have, and try again. If it's still happening, double check that the issue isn't persisting with REFramework removed.

**Reply by ExcessiveGBH** [reply]
Yes... all working now after deleting reframework.
Thanks for you help.
